Dockerfile

```diff
@@ -57,6 +57,6 @@ RUN chmod 0755 bin/*
 COPY --chown=ruby:ruby --from=build /usr/local/bundle /usr/local/bundle
 COPY --chown=ruby:ruby --from=build /app /app
 
-EXPOSE 3000
+EXPOSE 3002
```

Comment:
My ignorance about the deployment is making me nervous here.

Would a change be needed [in forms deploy](https://github.com/alphagov/forms-deploy/blob/e3a5cc6b2748c58e1f5cd417ab4621645b69eadb/infra/modules/forms-product-page/main.tf#L16)?

I'm not sure how it all gets connected - I think it boils down to [port mappings](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_PortMapping.html)

The API which uses a 9292 has a matching value in forms-deploy.

Reply:
Ah, good point. Interestingly forms-runner, which does have 3001 in the puma config , has 3000 in the Dockerfile and forms-deploy 🤔 I wonder how that works...

https://github.com/alphagov/forms-runner/blob/2f235b04ba8c41156a1523f48c1840063e82ee49/config/puma.rb#L18

https://github.com/alphagov/forms-runner/blob/2f235b04ba8c41156a1523f48c1840063e82ee49/Dockerfile#L66

https://github.com/alphagov/forms-deploy/blob/e3a5cc6b2748c58e1f5cd417ab4621645b69eadb/infra/modules/forms-runner/main.tf#L25

Reply:
So for reasons I don't understand changing the default port value in `config/puma.rb` works when running the app on my machine directly, but not when running the app in a Docker container. This seems like a bug 😬 Apparently there have been historical issues with Ruby and Puma tussling over what the default port should be, see https://github.com/rails/rails/issues/24435.

I'll leave the Dockerfile in this repo as `EXPOSE 3000` for now, but we might need to rethink our strategy in general?